src/beacon-api/app/routes/devices.py

```
@@ -505,4 +505,145 @@
+@router.get("/maintenance/overdue")
+async def get_overdue_maintenance_devices(
+    *,
+    db: Session = Depends(get_db)
+):
+    current_time = datetime.now(timezone.utc)
+    
+    # Get devices with overdue maintenance
+    devices = db.exec(
+        select(Device)
+        .where(
+            (Device.next_maintenance != None) & 
```

Ruff (0.12.2)

Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)